build_centos.dockerfile

```diff
@@ -0,0 +1,46 @@
+# https://github.com/ghdl/ghdl/issues/1484
+# https://github.com/ghdl/ghdl/issues/1118
+
+#
+# Centos 7
+#
+
+FROM centos:centos7
+
+RUN yum update -y && yum install -y git curl bzip2 wget flex fontconfig libX11 zlib-devel
+RUN yum install -y centos-release-scl && yum install -y devtoolset-8 texinfo
```

Comment:
It seems that the previous line (L10) is common for either centos7 or centos8. Please split this to a different stage.

Reply:
Done. Let me know if ok.